src/caching.js

```diff
@@ -53,7 +53,16 @@
       if (typeof(path) !== 'string') {
         throw new Error('path should be a string');
       }
-      if (typeof(value) === 'undefined') {
+      if (!RemoteStorage.util.isFolder(path)) {
+        //throw new Error('path should be a folder');
```

Comment:
I kind of forgot about this, but as said in #888 there are tests calling `Caching.set` with non-folder paths (trailing slash is missing). Should I just correct those tests?